gno.land/cmd/gnoweb/pages/HOME.md

```diff
@@ -1,50 +1,77 @@
-# Welcome to **Gno.land**
+# Welcome to Gno.land
 
-- [About Gno.land](/about)
+---
+
+## An interpretation of the Golang (Go) programming language for advanced developers and intrepid pioneers to build succinct, composable smart contracts for social coordination.
+
+If you’re concerned about information censorship and want to contribute to the #GnoWorldOrder, follow our socials to find out how
+
+Gno.land is in building mode. If you want to help lay the foundations of a fairer and freer world through innovative ideas and exceptional code, join us today.
+
+<div class="stack">
+
+[About Gno.land](/about)
+[Explore Github repo](https://github.com/gnolang)
+
+</div>
+
+<div class="jumbotron">
+
+### Game of Realms
+
+The first high-stakes contest will see participants compete for tiered membership to co-own the Gno.land blockchain. A series of complex technical and non-technical tasks will challenge contributors to create innovative patterns that push the chain to new limits. Start building the foundation for tomorrow through key smart contracts and other contributions that change our understanding of the world.
+
+<div class="stack">
+
+[Join Game Of Realms](/game-of-realms)
+
+</div>
+</div>
+
+<div class="columns-3">
+<div>
+
+### Learn about Gno.land
+
+- [About](/about)
+- [Subscribe](/)
```

Comment:
Shouldn't you add an anchor to the bottom of the page, where there is the subscribe form ?

Reply:
You are right!